lib/collection/src/shards/local_shard/mod.rs

```diff
@@ -889,9 +889,15 @@ impl LocalShard {
         filter: Option<&'a Filter>,
     ) -> CollectionResult<BTreeSet<PointIdType>> {
         let segments = self.segments().read();
+        let is_stopped = AtomicBool::new(false);
```

Comment:
I did not propagate further up as it was really noisy and the callers were not interested in cancellation anyway